### PR TITLE
Feat/when you click logo in menu you go to first menu item

### PR DESCRIPTION
**Pull request by mswertz**
Home icon will now go to first menu item instead of back to '/'
Includes org.jooq.Record pull request #276

### CODE REVIEW

**Comment by mswertz** [comment]
[When click logo I go to first menu item](https://trello.com/c/1YJi8pxR/114-when-click-logo-i-go-to-first-menu-item)

**Comment by connoratrug** [comment]
[Go to first menu item , on home click](https://trello.com/c/zj1M7zlv/119-go-to-first-menu-item-on-home-click)

File: apps/styleguide/src/layout/MolgenisMenu.vue [hunk 1]
```diff
@@ -71,6 +71,25 @@ export default {
...
+        let firstItem = this.permittedItems[0];
+        // in case it is a item with submenu and without href, find first submenu item
+        while (firstItem && firstItem.href == null && firstItem.submenu) {
+          let sub = firstItem.submenu.filter((i) => this.permitted(i));
+          firstItem = sub[0];
+        }
+        if (firstItem && firstItem.href) return firstItem.href;
+      }
+      //defaut: go home
+      return "../";
```
Comment (connoratrug) [inline apps/styleguide/src/layout/MolgenisMenu.vue:90]: why is the home '../'  ?

**Review by connoratrug — APPROVED** [review]

File: apps/styleguide/src/layout/MolgenisMenu.vue (hunk 1 above)
Comment (mswertz) [reply apps/styleguide/src/layout/MolgenisMenu.vue]: this automatically would lead you to default of the schema. Agree?

Comment (connoratrug) [reply apps/styleguide/src/layout/MolgenisMenu.vue]: yes , by why the '../' ( instead of '/' or '', or ../../, ...)

**Pull request merged**